include/ueye_cam/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+###############################################################################
```

Comment:
I understand the point of breaking up the top-level CMakeLists, but I'm not sure if having a couple of CMakeLists that only have 2 commands makes the top-level CMakeLists that much more readable. The ROS1 CMakeLists wasn't egregiously long. I think if I were to break-up the ROS1 version I would probably just split out the UEYE_DRIVER section.

Reply:
Getting into 'style' here. Some people prefer one borg-like blob. Others prefer it modular. Others a pragmatic in-between. 

I only broke it up here because I had to slowly wade through it all to get it working and find this mode easier to work with. Otherwise, I'm generally agnostic. Maybe a follow-up ticket and PR if it's important to you?